node/plan.sh

```diff
@@ -26,21 +26,13 @@ do_prepare() {
   fi
 }
 
-do_build() {
-  # use --without-snapshot, because https://github.com/nodejs/node/issues/4212
-  ./configure --prefix="$pkg_prefix" --without-snapshot
-  make
-}
-
 do_install() {
   do_default_install
 
   # Node produces a lot of scripts that hardcode `/usr/bin/env`, so we
   # need to fix that everywhere to point directly at the node binary. There is
   # no other good reason to add the env indirection trick on these shebangs.
-  grep -nrlI '^\#\!/usr/bin/env' "$pkg_prefix" | while read -r target; do
-    sed -e "s#\#\!/usr/bin/env node#\#\!${pkg_prefix}/bin/node#" -i "$target"
-  done
+  fix_interpreter "$pkg_prefix/*" core/coreutils bin/env
```

Comment:
This doesn't actually do anything. It needs to be something like:

    for f in grep -l -R ^#\!\/usr\/bin\/env "$pkg_prefix"; do
      fix_interpreter "$f" core/coreutils bin/env
    done